example/lib/main.dart

```diff
@@ -79,6 +92,19 @@ class _MyHomePageState extends State<MyHomePage> with TraceableClientMixin {
               '$_counter',
               style: Theme.of(context).textTheme.headlineMedium,
             ),
+            Padding(
+              padding: const EdgeInsets.symmetric(vertical: 20.0),
+              child: ElevatedButton(
+                onPressed: () async {
+                  await Navigator.of(context).push(MaterialPageRoute(
```

Comment:
You can directly use the shortcut `Navigator.push(context, ...)` for better readability.